content/ko/docs/tutorials/clusters/apparmor.md

```diff
@@ -0,0 +1,469 @@
+---
+reviewers:
+
+title: AppArmor
+content_template: templates/tutorial
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.4" state="beta" >}}
+
+
+AppArmor는 표준 리눅스 사용자와 그룹 기반의 권한을 보완하여, 한정된 리소스 집합으로
+프로그램을 제한하는 리눅스 커널 보안 모듈이다. AppArmor는 임의의 애플리케이션에 대해서
+잠재적인 공격범위를 줄이고 더욱 심층적인 방어를 제공하도록 구성할 수 있다.
+이 기능은 특정 프로그램이나 컨테이너에서 필요한 리눅스 기능, 네트워크 사용, 파일 권한 등에 대한
+접근 허용 목록를 조정한 프로파일로 구성한다. 각 프로파일은
+허용하지 않은 리소스 접근을 차단하는 *강제(enforcing)* 모드 또는
+위반만을 보고하는 *불평(complain)* 모드로 실행할 수 있다.
+
+AppArmor를 이용하면 컨테이너가 수행할 수 있는 작업을 제한하고 또는
+시스템 로그를 통해 더 나은 감사를 제공하여 더 안전한 배포를 실행할 수 있다.
+그러나 AppArmor가 은탄환(언제나 통하는 무적의 방법)이 아니며,
+애플리케이션 코드 취약점을 보호하기 위한 여러 조치를 할 수 있는 것 뿐임을 잊으면 안된다.
+양호하고 제한적인 프로파일을 제공하고, 애플리케이션과 클러스터를 여러 측면에서 강화하는 것이 중요하다.
+
+{{% /capture %}}
+
+{{% capture objectives %}}
+
+* 노드에 프로파일을 어떻게 적재 하는지 예시를 본다.
+* 파드에 프로파일을 어떻게 강제 적용하는지 배운다.
+* 프로파일이 적재되었는지 확인하는 방법을 배운다.
+* 프로파일을 위반하는 경우를 살펴본다.
+* 프로파일을 적재할 수 없을 경우를 살펴본다.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+다음을 보장해야 한다.
+
+1. 쿠버네티스 버전은 최소 1.4 이다. -- 쿠버네티스 v1.4부터 AppArmor 지원을 추가했다.
+   v1.4 이전 쿠버네티스 컴포넌트는 새로운 AppArmor 어노테이션을 인식하지 못하고
+   제공되는 AppArmor 설정을 **조용히 무시**할 것이다. 파드에서 예상하는 보호를 받고 있는지 확인하려면
+   해당 노드의 kubelet 버전을 확인하는 것이 중요하다.
+
+   ```shell
+   $ kubectl get nodes -o=jsonpath=$'{range .items[*]}{@.metadata.name}: {@.status.nodeInfo.kubeletVersion}\n{end}'
+   ```
+   ```
+   gke-test-default-pool-239f5d02-gyn2: v1.4.0
+   gke-test-default-pool-239f5d02-x1kf: v1.4.0
+   gke-test-default-pool-239f5d02-xwux: v1.4.0
+   ```
+
+2. AppArmor 커널 모듈을 사용 가능해야 한다. -- 리눅스 커널에 AppArmor 프로파일을 강제 적용하기 위해 AppArmor 커널 모듈은 반드시 설치되어 있고
+   사용 가능해야 한다. 예를 들어 우분투, SUSE 라든지 선택적으로 지원하는 대다수 서버 배포본에서는 기본적으로 모듈 사용한다.
```

Comment:
"예를 들어 우분투, SUSE 라든지 선택적으로 지원하는 대다수 서버 배포본에서는 기본적으로 모듈 사용한다." is a wrong translation from "Several distributions enable the module by default, such as Ubuntu and SUSE, and many others provide optional support.", since the translated statement interprets that "Ubuntu, SUSE, and many other optional support server distributions enable the module by default". The followings are my suggestions:

- "배포본" -> "배포판"
- Example suggestion of translation: "Ubuntu 및 SUSE와 같은 몇몇 배포판은 해당 모듈을 디폴트로 지원하며, 그 외 많은 다른 배포판들은 선택적으로 지원한다"

Reply:
🙆‍♂  반영했습니다.